### ISSUE TITLE
Projektek importálása és exportálása csv formátumban

### ISSUE THREAD

**Issue by matetamasi** [mok-it/mok-app#172]
Több projekt gyors létrehozására a ui használata helyett lehessen importált csv fájlt használni. A létrejövő projekt alapból tagokat nem tartalmaz.

A projekt tevékenységeinek kiértékeléséhez annak adatait exportálni is lehet, szintén csv formátumban. Mivel ilyenkor elsősorban a projekten végzett aktivitás az érdekes, exportálni kétféle csv-t is lehet. Az egyik csak a projekt adatait írja le, formátumában egyezik az importálható fájl elvárt formátumával. A másik exportálható fájl a projekt tagjait, és az általuk megszerzett mancsokat tartalmazza.

**Comment by notAnElephant** [comment]
#202  zárja? @matetamasi

**Comment by matetamasi** [comment]
Igen. Az export formátumon lenne mit finomítani, de ez is olyan featire amit értelmesebb esetben inkább weben használna az ember, szóval nem prio.
